docs/source/tutorials/interactive_simulation.rst

```diff
@@ -0,0 +1,185 @@
+- To go in depth and explore an Interactive simulation context, see <section>
+
+Setting up a Simulation
+=======================
+
+To run a simulation interactively, we will need to create a simulation object and furnish it with configurations,
+components and (optionally) plugins -- all the things that make up a simulation. The ``Vivarium`` framework provides
+four functions to help us get started with this, all found in ``vivarium_public_health``. They differ along two axes --
+how we give the simulation information about the components, plugins and configurations we'd like to simulate, and
+whether the simulation context is ``setup`` or not. However, each of these methods returns the same
+``InteractiveContext`` simulation object and can be interacted with in the same way.
+
+.. note:
+
+    see <section> for more on the difference between initialization and setup.
+
+The following examples will use the non-setup versions, but be aware that the counterparts exist that will call setup
+for you. Practically speaking, the utility of initializing without setting up is that it allows you to alter the
+configuration of a simulation before it is used. This is frequently useful for setting up specific configurations for
+validation from a notebook, for example. Changing the configuration at the top of the notebook makes it clear what is
+going on and guarantees that a configuration parameter is set as it should be.
+
+With a Model Specification File - The Automatic Way
+---------------------------------------------------
+
+The model specification file contains all the information needed to prepare and run a simulation so we need only
+provide this for one way to get up and running quickly.  We will use the model specification from our disease model
+examples:
+
+.. literalinclude:: ../../../src/vivarium/examples/disease_model/disease_model.yaml
+    :caption: **File**: :file:`disease_model.yaml`
+
+We can prepare and run a simulation interactively with this specification as follows. First, we initialize the
+simulation and get back an ``InteractiveContext`` object.
+
+.. code-block:: python
+
+    from vivarium_public_health.interactive import initialize_simulation_from_model_specification
+
+    p = "/path/to/disease_model.yaml"
+    sim = initialize_simulation_from_model_specification(p)
+
+In order to make it easier to follow along with this tutorial, We provide a convenience function to get the path to the
+disease model example specification distributed with Vivarium.
+
+.. code-block:: python
+
+    from vivarium_public_health.interactive import initialize_simulation_from_model_specification
+    from vivarium.examples.disease_model import get_model_specification_path
+
+    p = get_model_specification_path()
+    sim = initialize_simulation_from_model_specification(p)
+
+The function ``initialize_simulation_from_model_specification()`` returns a simulation object that has not been setup
```

Comment:
maybe use the function formatting e.g., `:func:`vivarium.interface.initialize_simulation_from_model_specification` because it then auto-links to the API docs